### DISCUSSION TITLE
[macos]: OS-friendly main menu

### DISCUSSION THREAD

**Discussion by zxcat** [doublecmd/doublecmd#1278] [Suggestions]
It would be great to have menu items in common places on macos. Every macos app have Application menu, which contains common items, including "About" and "Preferences".

When report bugs I have to open "About" dialog to get DC version, and out of habit I look for it in the **Application menu**, but it's in **Help menu**.
The same thing for Preferences, it's in App Menu on macos. (But many thanks to @rich2014 now I can open Prefs by simple `⌘,` shortcut which works now).

So idea is to customize menus on macos, moving to an App menu "About" and "Options"->"Preferences".

Here is an example implementation: [Mac Cross-Platform Challenges. Part 8: Making Your Lazarus App Mac-Friendly > Application menu](https://macpgmr.github.io/MacXPlatform/MacXPlatform_Part8.html#AppMenu).

**Comment by rich2014** [comment]
@alexx2000 hi, Alex

should we make DC consistent on each platform, or close to native APP on each platform?

**Comment by alexx2000** [comment]
Hi, it is a hard question. I can write like go to "Menu->Help->About" or "Menu->Configuration->Options" and it will be clear regardless of the platform. So I think it is better when DC is a consistent on each platform. 

But I think that we can add these two menu items to the App Menu too. So both methods will work.

**Comment by rich2014** [comment]
`About` and `Preferences` MenuItem added.

check the snapshoot please.
